src/components/Card/CardContent.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+
+export interface ICardContentProps {
+  className?: string;
+  children?: React.ReactChildren | React.ReactNode | string;
+}
+
+class CardContent extends React.PureComponent<ICardContentProps> {
+  public render() {
+    return <div className="Card__content padding-normal">{this.props.children}</div>;
```

Comment:
perhaps you meant to add this.props.className here?

Reply:
Good catch!